File: angular-web2-app/src/app/entities/incident/incident.ts
export class Incident {
    idIncidenta:number;
    nazivTipIncidenta:string;
    prioritet:number;
    potvrdjen:boolean;
    nazivStatusaIncidenta:string;
    ETA:Date;
    ATA:Date;
    ETR:Date;
    nivoNapona:number;
    planiranoVremeRada:Date;
    DatumKreiranja:Date;

    constructor(idIncidenta:number, nazivTipIncidenta:string, prioritet:number, potvrdjen:boolean, nazivStatusaIncidenta:string, ETA:Date,
        ATA:Date, ETR:Date, nivoNapona:number, planiranoVremeRada:Date, DatumKreiranja:Date) {
        this.idIncidenta = idIncidenta;
        this.nazivTipIncidenta = nazivTipIncidenta;
        this.prioritet = prioritet;
        this.potvrdjen = potvrdjen;
        this.nazivStatusaIncidenta = nazivStatusaIncidenta;
        this.ETA = ETA;
        this.ATA = ATA;
        this.ETR = ETR;
        this.nivoNapona = nivoNapona;
        this.planiranoVremeRada = planiranoVremeRada;
        this.DatumKreiranja = DatumKreiranja;
    }
}

File: api/webAPI/WebApplication1/Models/Incident.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models
{
    public class Incident
    {
        public string NazivTipIncidenta { get; set; }
        public int Prioritet { get; set; }
        public bool Potvrdjen { get; set; }
        public string NazivStatusaIncidenta { get; set; }
        public DateTime ETA { get; set; }
        public DateTime ATA { get; set; }
        public DateTime ETR { get; set; }
        public float NivoNapona { get; set; }
        public DateTime PlaniranoVremeRada { get; set; }
        public int IdKorisnika { get; set; } // osoba koja treba da resi incident, moze biti osoba koja je dodala incident ili nekome da se dodeli
    }
}